examples/platform/nrfconnect/util/include/FabricTableDelegate.h

```diff
@@ -42,19 +42,25 @@ class AppFabricTableDelegate : public chip::FabricTable::Delegate
 #ifndef CONFIG_CHIP_LAST_FABRIC_REMOVED_NONE
         static AppFabricTableDelegate sAppFabricDelegate;
         chip::Server::GetInstance().GetFabricTable().AddFabricDelegate(&sAppFabricDelegate);
+        k_timer_init(&sFactoryResetTimer, &FactoryResetTimerCallback, nullptr);
 #endif // CONFIG_CHIP_LAST_FABRIC_REMOVED_NONE
     }
 
 private:
     void OnFabricRemoved(const chip::FabricTable & fabricTable, chip::FabricIndex fabricIndex)
+    {
+        k_timer_start(&sFactoryResetTimer, K_MSEC(CONFIG_CHIP_LAST_FABRIC_REMOVED_SCHEDULE_TIME), K_NO_WAIT);
+    }
+
+    static void FactoryResetTimerCallback(k_timer * timer)
```

Comment:
nit (factory reset does not always happen):
```suggestion
    static void OnFabricRemovedTimerCallback(k_timer * timer)
```